content/documentation/device-examples/components/Servo.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Servo
+weight: 210
+tags: ["Servo"]
+---
+### Description
+The [Servo](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/example/components) (src/main/java/com/pi4j/example/components) is a template class, that you can use in your own Java-project.
+You can set the servo to a specific location, likewise to 110 degrees of it's range.
+
+You can use a wide variety of analog servo motors such as the SG92R or the SG-5010 (for a little more torque).
+
+### Layout
+![Servo Layout](/assets/documentation/device-examples/components/Layout-Servo.png)
+
+{{< gallery >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/ServoBreadboard.png" caption="Servo Breadboard" caption-position="center" caption-effect="fade" >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/ServoSG-5010-TopView.png" caption="Servo SG-5010 Top View" caption-position="center" caption-effect="fade" >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/ServoSG-5010-SideView.png" caption="Servo SG-5010 Side View" caption-position="center" caption-effect="fade" >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/ServoSG92R-SideView.png" caption="Servo SG92R Side View" caption-position="center" caption-effect="fade" >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/miniPowerSupplyModule.jpeg" caption="Mini Power Supply Module" caption-position="center" caption-effect="fade" >}}
+{{< figure link="/assets/documentation/device-examples/components/pictures/miniPowerSupplyModuleFrontBack.jpeg" caption="Mini Power Supply Module Front Back" caption-position="center" caption-effect="fade" >}}
+{{< /gallery >}}
+{{< load-photoswipe >}}
+
+### Code
+An example on how to use the Servo-Class from the [Hardware-Catalog](https://github.com/Pi4J/pi4j-example-components)
+
+```
+// Initialize servo motor component
+final var servoMotor = new ServoMotor(pi4j, PIN.PWM18.getPin());
```

Comment:
I see in preview, this code block doesn't format well. Maybe related to the final word here? 

Can you change ```   to ```java?

Reply:
i formatted the code-blocks on all components